cartesi-rollups/development/running-the-application.md

```diff
@@ -41,7 +41,7 @@ Attaching to 2bd74695-prompt-1, 2bd74695-validator-1
 
 This error happens when the node reads too many blocks behind the blockchain.
 
-To fix this, create a a `.sunodo.env.` and overwrite the set `TX_DEFAULT_CONFIRMATIONS` to 1, as in:
+To fix this, create a `.sunodo.env.` and overwrite the set `TX_DEFAULT_CONFIRMATIONS` to 1, as in:
 
 ```
 TX_DEFAULT_CONFIRMATIONS=1
```

Comment:
again, I don't like calling nonodo a node. the whole idea of the name is about running a cartesi backend with no node

Reply:
FIxed